linkedlist.py

```diff
@@ -0,0 +1,86 @@
+class Node:
+    def __init__(self, data=None, next=None):
+        self.data = data
+        self.next = next
+
+
+class LinkedList:
+    def __init__(self):
+        self.head = None
+
+    def show_list(self):
+        if self.head == None:
+            print('The list is empty')
+
+        current = self.head
+        linked_list = []
+        # while self.head is not None
+        while current:
+            linked_list.append(str(current.data))
+            current = current.next
+        print('-->'.join(linked_list))
+
+    def list_lenght(self):
```

Comment:
Generalnie fajnie, ze napisalas te metode w ten sposob dla treningu, ale wydaje mi sie, ze najlepszym rozwiazaniem byloby, zeby trzymac to jako propercje bezposrednio w klasie i trackowac kiedy dodajesz lub usuwasz element. W ten sposob bedziesz mogla latwiej robic jakies sprawdzenia w tych metodach z indexami bez zwiekszania zlozonosci obliczeniowej